conf/test.config

```
@@ -25,9 +25,10 @@ params {
     input                         = 'https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/samplesheet.csv'
     databases                     = 'https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/database.csv'
     run_kraken2                   = true
-    run_malt                      = true
+    run_malt                      = false
```

```suggestion
    run_malt                      = true
```

If you want to turn it off during testing, you can specify `-profile <container>,test --run_malt false` :)

---

Sorry, my bad :)